wled00/data/index.js

```diff
@@ -1696,6 +1696,8 @@ function setIntensity() {
 
 function setLor(i) {
 	var obj = {"lor": i};
+	// allow canceling live mode (if sender crashes)
+	if (i===0 && lastinfo.live && ["","Hyperion","UDP"].includes(lastinfo.lm)) { obj.live = false; obj.v = true; }
```

Comment:
I don't think this is a good idea, since `setLor(0)` is supposed to return to live data, not cancel it. It would likely be preferable to exit _indefinite_ (timeout == 65000) live mode on override (1 or 2), as turning off `lor` will not refresh the LEDs anyways.

Reply:
I know. I mentioned this on Discord. This "hack" allowed me to test cancelling live mode.
If you can figure out where to put `"live":false` and not obstruct UI I am happy with it.